src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,236 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    SemidiscretizationHyperbolicParabolic
+
+A struct containing everything needed to describe a spatial semidiscretization
+of a mixed hyperbolic-parabolic conservation law.
+"""
+
+struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, ParabolicEquations, InitialCondition,
```

Comment:
```suggestion
"""
struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, ParabolicEquations, InitialCondition,
```
Otherwise, the docstring is not attached to the `struct`.